phoenix-core/src/it/java/org/apache/phoenix/end2end/DerivedTableIT.java

```diff
@@ -54,14 +28,17 @@
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
 
-import com.google.common.collect.Lists;
+import java.sql.*;
+import java.util.*;
```

Comment:
Please see https://phoenix.apache.org/contributing.html#Code_conventions. You'll want to conform to these - easiest way is to import dev/eclipse_prefs_phoenix.epf and dev/phoenix.importorder into Eclipse. This is basically to ensure we don't get a lot of code differences due to different import ordering and whitespacing.